frontend/src/app/projects/project-detail/model-overview/model-overview.component.html

```diff
@@ -105,6 +105,14 @@ <h2>Models</h2>
           >
             <mat-icon>key</mat-icon>
           </a>
+          <button
```

Comment:
![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/c8a22e7b-16db-4bda-a4a3-f4d970ef8b6f)

The margins are not aligned properly.